### DISCUSSION TITLE
iOS Privacy manifests by Apple - Anything needed for mvvmcross ?

### DISCUSSION THREAD

**Discussion by viswanath-pamarthi** [MvvmCross/MvvmCross#4851] [General]
Hi @Cheesebaron ,

Is there a PrivacyInfo.xcprivacy file we need to include for MvvmCross & MvvmCross plugins for iOS App? 

Please let me know.

Please refer to below App Store requirement by Apple:
Upcoming third-party SDK requirements
https://developer.apple.com/support/third-party-SDK-requirements/
Privacy requirement for app submissions starts May 1
https://developer.apple.com/news/?id=pvszzano#:~:text=Starting%20May%201%2C%202024%2C%20new,Privacy%20manifests

https://devblogs.microsoft.com/dotnet/apple-privacy-manifest-support/

**Comment by entdark** [comment]
It is unrelated to MvvmCross.
https://devblogs.microsoft.com/dotnet/apple-privacy-manifest-support/

**Reply by viswanath-pamarthi** [reply]
Got it, Thanks for your response @entdark 
This is a requirement of Apple, so, wanted to make sure if none of the required reasons API are being used in the library.
